app/javascript/components/pages/JobsAuthenticate.jsx

```diff
@@ -1,36 +1,49 @@
 import React, { useState } from 'react';
+import { useCookies } from 'react-cookie';
 import SharedLayout from 'components/layout/SharedLayout';
 import Logo from 'components/icons/Logo';
 import Button from 'components/Button';
+import { postJobsAuthenticate } from '../../datasources';
 
 import 'stylesheets/page';
 import 'stylesheets/jobs_authenticate';
 
 const JobsAuthenticate = () => {
     const [password, setPassword] = useState('');
+    const [hasError, setHasError] = useState(false);
+    // eslint-disable-next-line no-unused-vars
+    const [_, setCookie] = useCookies(['token']);
+
+    const handleLogin = async (e) => {
+        e.preventDefault();
+
+        try {
+            const data = await postJobsAuthenticate(password);
+            setHasError(false);
+            setCookie('token', data.token);
```

Comment:
qq, I don't have a ton of context on how cookies work, but do you think `token` as the cookie name could be potentially overwritten by other websites? we could go with something more specific like `wnb_job_board_token`

Reply:
I don't either but you're right that it's better to be clear on what it might be.